com.getstation.Station.yaml

```diff
@@ -0,0 +1,50 @@
+id: com.getstation.Station
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
```

Comment:
current version is 24.08

Reply:
I know, but with 24.08 I have an error

```
Running appstream-compose
bwrap: execvp appstream-compose: No such file or directory
Error: ERROR: appstream-compose failed: Child process exited with code 1
```

Reply:
why do you run it? I don't see it in the manifest.

it was replaced by appstreamcli.

Reply:
I didn't get you comment. Flatpak a completely new area for me.
I build the package with the command:
```
flatpak-builder --install-deps-from=flathub --force-clean --user --install .build com.getstation.Station.yaml
```
and get the error with 24.08 but no error with 23.08
Do I use a wrong command?

Reply:
You likely use a version of flatpak-build that is too old.

See https://docs.flathub.org/docs/for-app-authors/submission#build-and-install on how to use the flatpaked version.

Reply:
yes. you were right!
it was old version. Thank you

updated to 24.08